ccpp/data/GFS_typedefs.F90

```diff
@@ -762,8 +762,12 @@ module GFS_typedefs
     !--- Thompson's microphysical parameters
     logical              :: ltaerosol       !< flag for aerosol version
     logical              :: lradar          !< flag for radar reflectivity
+    logical              :: fcons           !< flag for constraints
+    logical              :: ffixneg         !< flag for fixing negative traces
     real(kind=kind_phys) :: nsradar_reset   !< seconds between resetting radar reflectivity calculation
     real(kind=kind_phys) :: ttendlim        !< temperature tendency limiter per time step in K/s
+    real(kind=kind_phys) :: crtclw          !< ritical value for cloud water to rain in kg kg-1
```

Comment:
This should be "critical", not "ritical"

Reply:
yes. It should be "critical".